Explorer/Assets/DCL/Landscape/TerrainGenerator.cs

```diff
@@ -573,7 +585,195 @@ private void FreeMemory()
             noiseGenCache.Dispose();
         }
 
-        private static string TreeFilePath => $"{Application.streamingAssetsPath}/Trees.bin";
+        private static Texture2D CreateOccupancyMap(NativeArray<int2> emptyParcels, int2 minParcel,
+            int2 maxParcel, int padding)
+        {
+            int2 terrainSize = minParcel + maxParcel + 1;
```

Comment:
should it be `terrainSize = maxParcel  - minParcel + 1;` otherwise size will be 0 for (-151,-151) to (150,150) terrain

Reply:
Yeah, I just bumped into this while testing.